sormas-api/src/main/resources/captions.properties

```
@@ -289,6 +290,8 @@ Campaign.description=Description
 Campaign.startDate=Start date
 Campaign.endDate=End date
 Campaign.creatingUser=Creating user
+Campaign.deletionReason=Reason for deletion
+Campaign.otherDeleteReason=Reason for deletion details
```

```suggestion
Campaign.otherDeletionReason=Reason for deletion details
```

For all occurrences (since the field name has been changed)

---

changed